docs/dashboards/kpi-component.md

```diff
@@ -0,0 +1,96 @@
+KPI Component
+===
+
+The KPIComponent allows you to visualize key performance indicators.
+
+<iframe style="width: 100%; height: 470px; border: none;" src=https://www.highcharts.com/samples/embed/dashboards/components/component-kpi allow="fullscreen"></iframe>
+
+## How to start
+1. The KPIComponent is included in the Dashboard package, so nothing apart from the basic dashboards package needs to be imported.
+
+2. Define a cell using a unique identifier for example cell: `dashboard-col-0`.
+
+You can find more information how to create a layout in dashboard [here](https://www.highcharts.com/docs/dashboards/your-first-dashboard).
+
+3. The last thing that you have to do is to specify the `type: 'KPI'` in the component’s config and that’s it. See the full example below.
```

Comment:
You have to also add the `value: <value>` to display anything, so I wouldn't write it like that.